src/appkit.cs

```diff
@@ -5210,6 +5210,10 @@ interface NSDatePicker {
 		[Export ("maxDate", ArgumentSemantic.Copy)]
 		NSDate MaxDate { get; set; }
 
+		[NoiOS, Mac (10,15,4)]
```

Comment:
minor: you do not need `[NoiOS]` in `appkit.cs`

Reply:
```suggestion
		[Mac (10,15,4)]
```